Makefile

```diff
@@ -83,17 +83,7 @@ ifeq ($(TARGET_OS_LOCAL),windows)
 else
 	BUILD_TOOLS_BIN ?= build-tools
 	BUILD_TOOLS ?= ./.build-tools/$(BUILD_TOOLS_BIN)
-	RUN_BUILD_TOOLS ?= cd .build-tools; go run .
-endif
-
-ifeq ($(TARGET_OS_LOCAL),windows)
-	BUILD_TOOLS_BIN ?= build-tools.exe
-	BUILD_TOOLS ?= ./.build-tools/$(BUILD_TOOLS_BIN)
-	RUN_BUILD_TOOLS ?= cd .build-tools; go.exe run .
-else
-	BUILD_TOOLS_BIN ?= build-tools
-	BUILD_TOOLS ?= ./.build-tools/$(BUILD_TOOLS_BIN)
-	RUN_BUILD_TOOLS ?= cd .build-tools; go run .
+	RUN_BUILD_TOOLS ?= cd .build-tools; GOOS=$(TARGET_OS_LOCAL) GOARCH=$(TARGET_ARCH_LOCAL) go run .
```

Comment:
Is there a way to still allow using the pre-compiled ones? It's faster than recompiling with go run every time.

Reply:
Weren't the two blocks exactly identical? I thought that was unintentional, am I missing something? 🤔

Reply:
Uh, looks like the pre-compiled binaries weren’t actually used. oops :)

Reply:
Let's try to add it separately. Maybe using an environment variable that points to the binary? If missing, always use `go run .`. What do you think?

Reply:
That could be nice! But if we haven0t used it so far, maybe not necessary either.

Reply:
Yup, we will add it when it's needed then, thanks!